src/builtins/hmr-runtime.js

```diff
@@ -16,15 +16,14 @@ module.bundle.Module = Module;
 
 if (!module.bundle.parent) {
   var ws = new WebSocket('ws://localhost:{{HMR_PORT}}/');
-  ws.onmessage = (e) => {
-    var data = JSON.parse(e.data);
+  ws.onmessage = function(event) {
+    var data = JSON.parse(event.data);
+    var key, asset;
 
     if (data.type === 'update') {
-      for (let asset of data.assets) {
+      for (key in data.assets) {
```

Comment:
not a good idea to use a `for..in` loop over an array. can you either make this a normal `for` loop or `data.assets.forEach(function (asset) { ... })`?

Reply:
yep, i'll do that now.